app/controllers/miq_ae_tools_controller.rb

```diff
@@ -89,16 +89,27 @@ def import_export
     drop_breadcrumb(:name => "Import / Export", :url => "/miq_ae_tools/import_export")
     @lastaction = "import_export"
     @layout = "miq_ae_export"
-    @importable_domain_options = [["<Same as import from>", nil]]
+    @importable_domain_options = []
     MiqAeDomain.all_unlocked.collect do |domain|
       @importable_domain_options << [domain.name, domain.name]
     end
+
+    editable_domains = current_tenant.editable_domains.collect(&:name)
+    @importable_domain_options = @importable_domain_options.select do |importable_domain|
+      editable_domains.include?(importable_domain[0])
+    end
+
+    @importable_domain_options.unshift(["<Same as import from>", nil])
     render :action => "show"
   end
 
   def automate_json
     begin
-      automate_json = automate_import_json_serializer.serialize(ImportFileUpload.find(params[:import_file_upload_id]))
+      editable_domains = current_tenant.editable_domains.collect(&:name)
+      automate_json = automate_import_json_serializer.serialize(
+        ImportFileUpload.find(params[:import_file_upload_id]),
+        editable_domains
```

Comment:
`automate_import_json_serializer.serialize` call is raising error `wrong number of arguments (2 for 1)`.  You do not need to pass in the `editable_domains` here so this code can be reverted to the original logic.